aiohttp/helpers.py

```diff
@@ -356,14 +357,40 @@ def parse_mimetype(mimetype: str) -> MimeType:
     )
 
 
+class EnsureOctetStream(Message):
+    def __init__(self) -> None:
+        super().__init__()
+        self.set_default_type("application/octet-stream")
+
+    def get_content_type(self) -> Any:
+        """Re-implementation from Message
+
+        Returns application/octet-stream in place of plain/text when
+        value is wrong.
+
+        The way this class is used guarantees that content-type will
+        be present so simplify the checks wrt to the base implementation.
+        """
+        value = str(self.get("content-type")).lower()
```

Comment:
Would this not always be a string, if we change the default?
```suggestion
        value = self.get("content-type", "").lower()
```

Reply:
Good point, it should indeed.